blog/MeetthehoodiesNickHehr.md

```diff
@@ -0,0 +1,33 @@
+---
+
+title: 'Meet the Hoodies: Nick Hehr'
+author: Pawan
+layout: blog-post
+comments: true
+
+---
+
+#Hello all! I recently had the great honor of interviewing Nick Hehr ([Twitter](https://twitter.com/hipsterbrown), [GitHub](https://github.com/hipsterbrown)). Let's see what he had to say:
+
+#1. What’s your favorite animal?
```

Comment:
you need to add a space after the `#`, also can we make this a 2nd grade header with `## `? Sorry again, I should have catched that before. If you could update the headings then it’s good to go